mapactionpy_arcmap/LayerProperty.py

```diff
@@ -1,14 +1,15 @@
 from LabelClass import LabelClass
 
+
 class LayerProperty:
     def __init__(self, row):
-       self.mapFrame = row["MapFrame"]
-       self.layerGroup = row["LayerGroup"]
-       self.layerName = row["LayerName"]
-       self.sourceFolder = row["SourceFolder"]
-       self.regExp = row["RegExp"]
-       self.definitionQuery = row["DefinitionQuery"]
-       self.display = row["Display"]
-       self.labelClasses = list()
-       for labelClass in row["LabelClasses"]:
-           self.labelClasses.append(LabelClass(labelClass))
+        self.mapFrame = row["MapFrame"]
+        self.layerGroup = row["LayerGroup"]
+        self.layerName = row["LayerName"]
+        self.sourceFolder = row["SourceFolder"]
```

Comment:
It is possible to simplify this class, but more significantly the `layerProperties.json` file. The `sourceFolder` is not required. The DNC requires datanames are unique throughout the `2_Active_Data` folder. This was in part to support moving the entire `2_Active_Data` folder to a geodatabase at some point. The subfolders are purely there for human browsing. 

Using `os.walk()` simplifies the process of constructing the path for the shapefiles. For example here:
https://github.com/mapaction/mapactionpy_controller/blob/master/mapactionpy_controller/data_search.py#L31

It would be appropriate to have a check that the data files within `2_Active_Data` _are_ unique and that they are in the correct subfolder. If they are not then this indicates that there was an error in the data scramble process. However this check could/should exist outside the MapChef tool. eg there is the beginning of such a tool here:
https://github.com/mapaction/mapactionpy_controller/pull/15/files#diff-d6f40ae1e4497253650bdc577bf12bdc